src/containers/Detail/styles.module.scss

```diff
@@ -0,0 +1,64 @@
+@import '@scss/mixins';
+
+.cardHeader {
+  padding: 1.3rem;
+  margin-bottom: 0;
+  border-bottom: 1px solid rgba(222, 226, 230, 70%);
+}
+
+.mt10 {
+  margin-top: 10px;
+}
+
+h2 {
+  margin-left: 1.3rem;
+}
+
+.cardHeader:first-child {
+  border-radius: 0.25rem 0.25rem 0 0;
+}
+
+.calendarIcons {
+  font-size: 24px;
+}
+
+b {
+  font-weight: 600px;
```

Comment:
This can be delete if using `Mantine Text` component. But one thing to remember: The font weight property's value do not ship with any unit.

For more information: [font-weight](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight)

Reply:
done